README.md

```diff
@@ -93,44 +93,48 @@ The following apps are supported by govuk-docker to some extent.
    - ⚠ cache-clearing-service
       * Tests pass
       * Queues are not set-up, so cache-clearing-service can't be run locally
-   - ✅ calendars
    - ⚠ calculators
       * Web UI doesn't work without the content item being present in the content-store.
+   - ✅ calendars
+   - ⚠  collections
+      * You will need to [populate the Content Store database](#mongodb) or run the live stack in order for it to work locally.
+      * To view topic pages locally you still need to use the live stack as they rely on Elasticsearch data which we are yet to be able to import.
    - ✅ collections-publisher
    - ⚠ content-data-admin
       * **TODO: Missing support for a webserver stack**
    - ✅ content-publisher
    - ✅ content-store
    - ✅ content-tagger
-   - ⚠  collections
-      * You will need to [populate the Content Store database](#mongodb) or run the live stack in order for it to work locally.
-      * To view topic pages locally you still need to use the live stack as they rely on Elasticsearch data which we are yet to be able to import.
    - ✅ email-alert-api
    - ✅ email-alert-frontend
    - ✅ finder-frontend
    - ❌ frontend
    - ✅ government-frontend
    - ✅ govspeak
+   - ✅ govuk_app_config
+   - ✅ govuk_publishing_components
    - ✅ govuk-cdn-config
+   - ⚠ govuk-content-schemas
+      * Pending https://github.com/alphagov/govuk-docker/issues/173
```

Comment:
I'm not sure this one is really an issue with this particular service.

Reply:
Fair point 👍 can't quite remember why I linked this issue with this particular service.

Reply:
Not quite sure what to put in its absence. Reluctant to put a `✅` without _knowing_ it works, and because of issue #173 I'm unable to verify that.

Thinking of putting this, unless you can put my mind at rest?

```
   - ❓ govuk-content-schemas
      * Service exists in govuk-docker but is untested
```

Reply:
Works for me :thumbsup: